### ISSUE TITLE
Logo breaks layout with increased font size

### ISSUE THREAD

**Issue by robinwhittleton** [alphagov/govuk_template#232]
WCAG2 1.4.4 says that to hit double-AA [text must be able to zoom to 200% of the default size without breaking the site layout](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html). Currently the logo is fixed to 30px which breaks the text when resized:

![screen shot 2016-07-14 at 15 27 35](https://cloud.githubusercontent.com/assets/7414/16843029/87665130-49d7-11e6-8d54-a5f75d2d5514.png)

We should allow the header to grow in size to contain the text, probably by using `min-height` instead of `height`. Reported by @selfthinker.

**Comment by accessiblewebuk** [comment]
This isn't a WCAG 2 failure in itself, because the text isn't obscured and the contrast between the white text and the blue background is still sufficient. Obviously it would be best practice for the layout not to break.

**Comment by nickcolley** [comment]
Fixed by #237 thanks @selfthinker 🎉
